Project-Hotpot/Project-Hotpot/ViewControllers/ViewController.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  ViewController.swift
+//  Project-Hotpot
+//
+//  Created by Harini Sundaram on 7/26/22.
+//
+
+import UIKit
+//for keeping general functions to be accessed by all view controllers
+class ViewController: UIViewController {
```

Comment:
Good job creating a superclass to make stuff reusable. Nit: "ViewController" is a very generic name. Could be something like HotpotViewController or BaseViewController.